docs/catalog/sinks/RobustaUI.rst

```diff
@@ -0,0 +1,44 @@
+Robusta UI
+#################
+
+Robusta can send Playbooks results to Robusta UI.
```

Comment:
Can we phrase this without "Playbook results" in a way that makes sense to users unfamiliar with Robusta?

E.g. "Robusta can send Kubernetes errors and alerts to the Robusta UI."

Then include a screenshot or gif of it up top. Show why you should use this, before how.